src/clj/chronograph/db/organization.clj

```diff
@@ -15,3 +16,24 @@
                    :created-at now
                    :updated-at now}
                   db/sql-opts))))
+
+
+(defn find-one
```

Comment:
I'm not comfortable with doing this. 

I don't think it's a good idea to put argument information in the function name --- it makes the function too specific and adds overhead of having to remember to revise the function name if argument names change.

I think `(find-one [slug user-id] ...)` reads much better, and also suggests a pattern general to other "find-one" situations.

Besides, IDEs are quite good at displaying arg lists and documentation.

Reply:
In which case maybe we should call it `find` and it's arguments can be a map, so you can use the same function for find by id, etc.

Reply:
`find-one` is a deliberate choice. `find` can potentially return any number of items, whereas `find-one` suggests it will return exactly one item, which is what we require here.

Both semantics are meaningful and will be useful to us e.g. find users/tasks/orgs/things matching certain criteria v/s find exactly this org/user/task/thing.

Whether args should be a map depends on the number of args and optionality of args. Here we have only two, and both are necessary. So it is better to have a function api that explicitly requires both things. If we use a map, then we will need boilerplate checks to make sure both values are provided.

More generally, I think it makes sense to agree upon a common standard to make these choices. We'll bike-shed a lot otherwise.

Reply:
Related: Stuart Sierra presents an argument about avoiding [Heisenparameters](https://stuartsierra.com/2015/06/10/clojure-donts-heisenparameter), suggesting function names such as "process-one" / "process-batch" to suggest argument type (item? or coll?). 

We would like to avoid the corresponding problem of---let's call it Heisenreturns---and suggest that with "find-one" v/s "find".

Reply:
Will fix name / semantics based on the discussion about separating out ACL check from this "find one org" logic.

Reply:
Done. Refactored the logic.